### PR TITLE
Fix rebatch bar

### PR DESCRIPTION
**Pull request by AlexeyKozhevin**
* `Notifier` will work with `NE` arguments in `prefetch` mode
* Now bar will correctly work with rebatch, `total` argument is needed.

### CODE REVIEW

**Comment by roman-kh** [comment]
This will conflict with #504

**Comment by roman-kh** [comment]
@AlexeyKozhevin  Fix conflicts, please

**Comment by roman-kh** [comment]
@AlexeyKozhevin

**Comment by roman-kh** [comment]
@AlexeyKozhevin Fix the conflict or delete the PR

**Comment by AlexeyKozhevin** [comment]
Update PR:

- add `total` parameter for the number of resulting batches if you are using `rebatch`
- fix `Notifier` update in case of using `prefetch`

File: batchflow/pipeline.py [hunk 1]
```diff
@@ -1510,6 +1510,7 @@ def _gen_batch(self, *args, **kwargs):
         prefetch = kwargs.pop('prefetch', 0)
         on_iter = kwargs.pop('on_iter', None)
         notifier = kwargs.pop('notifier', kwargs.pop('bar', None))
+        total = kwargs.pop('total', None)
```
Comment (roman-kh) [inline batchflow/pipeline.py:1513]: Let's issue a deprecated warning if `bar` is passed.

Comment (roman-kh) [reply batchflow/pipeline.py:1513]: Please, also fix notifier init - it creates DummyBar on else clause. The better behaviour would be to create DummyBar on None and issue a warning for unknown  value.

**Review by roman-kh — CHANGES_REQUESTED** [review]

**Review by SergeyTsimfer — APPROVED** [review]

**Pull request merged**